docs/docs/pages/docs/doc.js

```diff
@@ -4,7 +4,7 @@ function Doc (props) {
   return (
     <div className='docs-doc'>
       <div className='docs-doc-edit'>
-        <a href={`https://github.com/cerebral/cerebral/edit/master/docs/docs/${props.docName}.md`} target='_new'>Edit on Github</a>
+        <a href={`https://github.com/cerebral/cerebral/edit/master/docs/docs/docs/${props.sectionName}/${props.docName}.md`} target='_new'>Edit on Github</a>
```

Comment:
Replace with regular link, not edit one. Edit link is usable only for members of repo.